buildscripts/cmake/DeclareModuleSetup.cmake

```diff
@@ -65,6 +65,17 @@ macro(declare_module name)
 endmacro()
 
 
+macro(add_qml_import_path input_var)
+  if (NOT ${${input_var}} STREQUAL "")
+      set(QML_IMPORT_PATH "$CACHE{QML_IMPORT_PATH}")
+      list(APPEND QML_IMPORT_PATH ${${input_var}})
+      list(REMOVE_DUPLICATES QML_IMPORT_PATH)
```

Comment:
Since we can't be sure `${${input_var}}` isn't a list (multiple paths), it isn't safe to use `list(FIND)` to avoid duplication. For example:

```cmake
set(_new_paths /a/b /a/c)
set(QML_IMPORT_PATH /a/c)
list(FIND QML_IMPORT_PATH "${_new_paths}" _exists)
if (_exists EQUAL -1)
  list(APPEND QML_IMPORT_PATH ${_new_paths})
endif()
```
...will set `QML_IMPORT_PATH` to `/a/c;/a/b;/a/c`.